tests/e2e/tests/electron/critical-path/a-first-start-form/autodiscovery.e2e.ts

```diff
@@ -11,12 +12,17 @@ const databaseAPIRequests = new DatabaseAPIRequests();
 const standalonePorts = [8100, 8101, 8102, 8103, 12000];
 const otherPorts = [28100, 8200];
 
+//TODO Other tests are messing up the state of this test for now seams like deleting the folder  will do as workaround
+const cleanBeforeClientStarts = async () => {await Common.deleteFolderIfExists(workingDirectory)}
+cleanBeforeClientStarts().then(() => {})
+
 fixture `Autodiscovery`
     .meta({ type: 'critical_path', rte: rte.none })
     .page(commonUrl)
-    .beforeEach(async() => {
-        await databaseHelper.acceptLicenseTerms();
+    .beforeEach(async(t) => {
+         await databaseHelper.acceptLicenseTerms();
```

Comment:
```suggestion
        await databaseHelper.acceptLicenseTerms();
```
not sure tho, I just saw an empty space and I'm wondering if should be there; maybe it wasn't formatted OK in the first place